invenio_pidstore/ext.py

```diff
@@ -26,34 +26,42 @@
 
 from __future__ import absolute_import, print_function
 
-from . import config
+import pkg_resources
+
 from .models import PersistentIdentifier
+from .errors import PIDDoesNotExistError
 
 
-def pid_exists(value, pidtype="doi"):
+def pid_exists(value, pidtype=None):
     """Check if a persistent identifier exists."""
-    return PersistentIdentifier.get(pidtype, value) is not None
+    try:
+        PersistentIdentifier.get(pidtype, value)
+        return True
+    except PIDDoesNotExistError:
+        return False
 
 
 class InvenioPIDStore(object):
     """Invenio-PIDStore extension."""
 
-    def __init__(self, app=None):
+    def __init__(self, app=None, entry_point='invenio_pidstore.minters'):
         """Extension initialization."""
+        self.minters = {}
+        self.entry_point = entry_point
```

Comment:
IMHO it's better patter to pass `**kwargs` (`__init__(app=None, **kwargs)`) to `init_app(..., entrypoint_name='invenio_pidstore.minters')` on line 52 keeping it as close to the place where it's used as possible.